client/test/Signin.test.tsx

```diff
@@ -0,0 +1,21 @@
+/* eslint-disable no-console */
+// currently not working :)
+import { render, screen } from '@testing-library/react';
```

Comment:
We are doing tests using playwright framework. Please move these tests to integration test templates used in the existing code base.

The "@testing-library/react'" and "@testing-library/jest-dom" libraries are a perfect fit for testing pure react components. @DaloonOfDoom @Omarley7 @SyntaxXeror   please take a note of this.